preprocessy/handlenullvalues/_main.py

```diff
@@ -138,6 +167,14 @@ def execute(self):
             and self.fill_missing is None
         ):
             self.final_df = self.__fill_values_columns()
+        
+        elif (
+            self.drop is not None
+            and self.drop != False
```

Comment:
`self.drop is not None and self.drop != False` implies `self.drop == True`. You can combine them into a single condition.